torchio/transforms/preprocessing/intensity/clamp.py

```diff
@@ -27,6 +27,13 @@ def __init__(
         super().__init__(**kwargs)
         self.out_min, self.out_max = out_min, out_max
 
+    def __repr__(self):
```

Comment:
The parent class already has this method. And it would seem a bit arbitrary to have only this one transform have a custom `repr`.

Reply:
Should I add the `args_names` attribute to the `Clamp` class so the `repr` looks like how I had it previously?

Reply:
I just did that, thanks. The main importance of `args_names` (which is not well documented and has a bad variable name) is to be able to reproduce the transform applied to a subject, so it was indeed necessary. Good catch!